src/components/SettingsPage/ExperimentalSwitch.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import React, { PureComponent } from 'react'
+
+import Track from 'analytics/Track'
+import Switch from 'components/base/Switch'
+
+type Props = {
+  name: string,
+  valueOn?: mixed,
+  valueOff?: mixed,
```

Comment:
remove the two `?:` and define a `defaultProps = { valueOn: true, valueOff: false }` . that would simplify the logic inside `onChange`